content/en/docs/instrumentation/java/springboot.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Spring Boot
+linkTitle: Spring Boot
+aliases:
+  - /docs/java/getting_started
+  - /docs/java/springboot
+  - /docs/instrumentation/java/springboot
+weight: 30
+description: Spring instrumentation for OpenTelemetry Java
+# prettier-ignore
+---
+
+You can instrument a Spring Boot application with the [OpenTelemetry Java agent](automatic/_index).
+
+Another option is to use the OpenTelemetry Spring Boot starter. The rest of this page document it.
+
+## Getting Started
+
+OpenTelemetry Spring Starter is a starter package that allow you to setup distributed tracing.
+
+Check out [opentelemetry-spring-boot-autoconfigure](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/spring/spring-boot-autoconfigure/README.md#features) for the full list of supported libraries and features.
+
+It is compatible with Spring Boot 2.0, Spring Boot 3.0 and Spring native.
+
+Add the dependency given below in your project to enable the OpenTelemetry Spring Starter.
+
+You have to replace`OPENTELEMETRY_VERSION` with the latest stable [release](https://search.maven.org/search?q=g:io.opentelemetry).
```

Comment:
```suggestion
You have to replace`OPENTELEMETRY_VERSION` with the latest stable [release](https://search.maven.org/search?q=g:io.opentelemetry).
```

If the starter is in sync with other releases we should be able to pull in the latest version like we do it on the other pages.

Reply:
It is fixes with 125a3823e30186a39d5bca9ec0437f928591b1cc